### ISSUE TITLE
Error when trying to connect to server

### ISSUE THREAD

**Issue by rydens** [brainfoolong/web-ftp-client#6]
This system is awesome! Exactly what I've been looking for. As far as I can tell, this is connecting to my remote server, but none of the files ever show up, and when I hit "Connect", I get this message that pops up:
<img width="429" alt="screen shot 2017-10-12 at 9 51 51 am" src="https://user-images.githubusercontent.com/12144736/31501185-838ff11c-af37-11e7-92d4-08939c36862b.png">

When I hit "Connect" after filling in my server info, this red box pops up and the server files don't show.

Thanks,
rydens

**Comment by brainfoolong** [comment]
Seems like that your server data is not properly saved. Can you delete the server from your list and than add the server again?

**Comment by rydens** [comment]
Thanks for the quick reply, and sorry for my late one. I discovered that the issue was I had not put my password for my remote server into the configurations area, assuming that I would be prompted for my password. I am too used to other clients giving me the option of not saving my passwords; maybe that could be a suggestion?

Anyhow, sorry for any time I wasted, and thanks again for the help.

\- rydens
